Assets/_BuildingGame/Scripts/Player/PlayerMovement.cs

```diff
@@ -0,0 +1,80 @@
+using BuildingGame.Core;
+using UnityEngine;
+
+namespace BuildingGame.Player
+{
+    public class PlayerMovement : BaseBehaviour
+    {
+        private const float walkingSpeed = 7.5f;
+        private const float runningSpeed = 11.5f;
+        private const float jumpSpeed = 8.0f;
+        private const float gravity = 20.0f;
+        private Camera playerCamera;
+        private const float lookSpeed = 2.0f;
+        private const float lookXLimit = 45.0f;
```

Comment:
This can be replaced by the new Behaviour Data system that uses scriptable objects to store the mutable data and can be retrieved by the scratchpad called the LostAndFound. That way you don't have to use magic numbers.